grammars/shell-unix-bash.cson

```diff
@@ -21,7 +21,7 @@
   'cygport'
   'bats'
 ]
-'firstLineMatch': '^#!.*\\b(bash|zsh|sh|tcsh|ksh|dash|ash|csh)|^#\\s*-\\*-[^*]*mode:\\s*shell-script[^*]*-\\*-'
+'firstLineMatch': '^#!.*\\b(bash|zsh|sh|tcsh|ksh|dash|ash|csh|rc)|(?i)(?:^\\s*#\\s*-\\*-[^*]*mode:\\s*shell-script[^*]*-\\*-)'
```

Comment:
Don't worry, it doesn't affect the rest of the expression. I tested it with `#!/BIN/SH` and it didn't match. =)